06LessonSix/in-class-project/css/intel-style.css

```diff
@@ -1,25 +1,25 @@
 
 body {
-    font-family: 'Arial', sans-serif;
+    font-family: 'Arial';
 }
 
-header, main, footer {
+header main footer {

```

Comment:
what was wrong with having a comma `,` after header and main?
these selectors where fine.